src/Sound/Tidal/Params.hs

```diff
@@ -179,6 +179,18 @@ end = pF "end"
 gain :: Pattern Double -> ControlPattern
 gain = pF "gain"
 
+-- | 'vol' is a version of gain with a built-in limiting that clips the provided
+-- values to stay between 0 and 1.999 in order to protect against common
+-- typing errors causing dangerously or uncomfortably loud sounds.
+vol :: Pattern Double -> ControlPattern
+vol = gain . clip 0 1.999
```

Comment:
Why not implement this in the target (e.g. scdirt/webdirt) rather than in tidal?
@telephon do you have a view on this?

Reply:
One could just as well ask why not here? Either way you have to change something in Tidal because both cases - safe and unsafe - need to be available to people who are live coding.

Reply:
It's a choice between changing it in one place versus having to change it in three places. But I don't care too much how it gets implemented (we're talking about real small change of efficiency anyway) so long as the result is something safer for everyone.